packages/databyss-ui/components/PageContent/IndexPageContent.js

```diff
@@ -1,9 +1,18 @@
 import React from 'react'
 import { Text, View, ScrollView } from '@databyss-org/ui/primitives'
-import { pxUnits } from '@databyss-org/ui/theming/views'
+import { pxUnits, widthVariants } from '@databyss-org/ui/theming/views'
 
 const IndexPageContent = ({ title, children }) => (
-  <ScrollView p="medium" flex="1" maxHeight="98vh">
+  <ScrollView
+    p="medium"
+    flex="1"
+    maxHeight="98vh"
+    maxWidth={widthVariants.content.maxWidth}
+    css={{
+      overflowWrap: 'anywhere',
+      wordBreak: 'break-word',
+    }}
```

Comment:
Let's add these styles to a `wrapVariant` on all `View` and derived components

Reply:
That makes sense, should the overflowWrap: 'anywhere' be the default then?